be/src/exec/vectorized/json_scanner.cpp

```diff
@@ -781,19 +733,19 @@ Status JsonArrayParser::get_current(simdjson::ondemand::object* row) {
         simdjson::ondemand::value val = *_array_itr;
 
         if (val.type() != simdjson::ondemand::json_type::object) {
-            return Status::DataQualityError("nested array is not supported");
+            return Status::DataQualityError("json object is expected");
```

Comment:
This error message is less accurate than the previous one

Reply:
Updated. The content of the value is added.